examples/src/croncat.rs

```diff
@@ -0,0 +1,204 @@
+// This example will go over the croncat contracts found
+// [here](https://github.com/CronCats/contracts/blob/cafd3caafb91b45abb6e811ce0fa2819980d6f96/manager/src/lib.rs)
+// This will demonstrate a more involved example for fast-forwarding the blockchain to a future
+// state/time. This is useful for testing anything that is time dependent such as scheduling
+// This is perfect to showcase cron.cat which will schedule calling into contract functions
+// at a set amount of time we supply.
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use near_sdk::json_types::U128;
```

Comment:
prob shouldn't import and compile the SDK for just this and serde, wonder if there is a better alternative for this. Guess it doesn't have to come with this, but avoiding linking to SDK for examples is probably best

Reply:
Probably then we should provide our own set of these `U128`/`U64` types so people don't have to import SDK directly to make use of them. I'll put this in a separate PR

Reply:
Those types are not the best to interact with. It would be best if there was a convenience module you can use in an attribute rather than using wrapper types.